<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Second</class>
 <widget class="QWidget" name="Second">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>950</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>950</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1100</width>
    <height>950</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>313</x>
     <y>10</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>长度转换</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_6" native="true">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>77</y>
     <width>699</width>
     <height>50</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="lineEdit_9">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>4</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_1">
    <property name="geometry">
     <rect>
      <x>236</x>
      <y>4</y>
      <width>60</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>米</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>英尺</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>千米</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>丈</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="equal1">
    <property name="geometry">
     <rect>
      <x>303</x>
      <y>-2</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_10">
    <property name="geometry">
     <rect>
      <x>403</x>
      <y>4</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>581</x>
      <y>4</y>
      <width>60</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>米</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>英尺</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>千米</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>丈</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>142</y>
     <width>761</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_1">
      <property name="text">
       <string>其它-》公制</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_11">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>其它-》公制</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_1</zorder>
   <zorder>horizontalSpacer</zorder>
   <zorder>horizontalSpacer_2</zorder>
   <zorder>horizontalSpacer_11</zorder>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>333</x>
     <y>210</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>重量转换</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>274</y>
     <width>699</width>
     <height>50</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>4</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>236</x>
      <y>1</y>
      <width>60</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>50</height>
     </size>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>斤</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>磅</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>千克</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="equal2">
    <property name="geometry">
     <rect>
      <x>303</x>
      <y>8</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_8">
    <property name="geometry">
     <rect>
      <x>403</x>
      <y>4</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_4">
    <property name="geometry">
     <rect>
      <x>581</x>
      <y>1</y>
      <width>60</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>50</height>
     </size>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>斤</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>磅</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>千克</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_8" native="true">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>330</y>
     <width>521</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer_13">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>其它-》公制</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_12">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>180</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>其它-》公制</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_14">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>343</x>
     <y>390</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>面积转换</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>445</y>
     <width>759</width>
     <height>50</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>4</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_5">
    <property name="geometry">
     <rect>
      <x>236</x>
      <y>4</y>
      <width>90</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>平方米</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>英亩</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>公顷</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>平方千米</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="equal3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>433</x>
      <y>4</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_6">
    <property name="geometry">
     <rect>
      <x>611</x>
      <y>4</y>
      <width>90</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>平方米</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>英亩</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>公顷</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>平方千米</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_9" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>721</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <spacer name="horizontalSpacer_15">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string>其它-》公制</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_23">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>180</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_6">
      <property name="text">
       <string>其它-》公制</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_17">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
   <zorder>pushButton_6</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>horizontalSpacer_15</zorder>
   <zorder>horizontalSpacer_17</zorder>
   <zorder>horizontalSpacer_23</zorder>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>570</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>体积转换</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>621</y>
     <width>777</width>
     <height>50</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>4</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_7">
    <property name="geometry">
     <rect>
      <x>236</x>
      <y>4</y>
      <width>99</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>立方米</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>毫升（ml）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>英国加仑</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>升（L）</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="equal4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>442</x>
      <y>4</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_8">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>4</y>
      <width>99</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>立方米</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>毫升（ml）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>英国加仑</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>升（L）</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_10" native="true">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>686</y>
     <width>811</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <spacer name="horizontalSpacer_16">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>其它-》公制</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_22">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>其它-》公制</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_18">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>760</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>温度转换</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>790</y>
     <width>775</width>
     <height>61</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="lineEdit_1">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>14</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_9">
    <property name="geometry">
     <rect>
      <x>236</x>
      <y>14</y>
      <width>98</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>摄氏度°C</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>开式度°K</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>华氏度°F</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="equal5">
    <property name="geometry">
     <rect>
      <x>341</x>
      <y>11</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>441</x>
      <y>14</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_10">
    <property name="geometry">
     <rect>
      <x>619</x>
      <y>14</y>
      <width>98</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>摄氏°C</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>开式度°K</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>华氏°F</string>
     </property>
    </item>
   </widget>
   <zorder>comboBox_10</zorder>
   <zorder>comboBox_9</zorder>
   <zorder>lineEdit_1</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>horizontalSpacer_9</zorder>
   <zorder>horizontalSpacer_10</zorder>
   <zorder>equal5</zorder>
  </widget>
  <widget class="QWidget" name="widget_11" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>866</y>
     <width>781</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <item>
     <spacer name="horizontalSpacer_19">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_9">
      <property name="text">
       <string>其它-》公制</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_21">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_10">
      <property name="text">
       <string>其它-》公制</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_20">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>750</y>
     <width>1001</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>560</y>
     <width>1001</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>380</y>
     <width>1001</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>1001</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
